frontend/src/scenes/insights/Insights.tsx

```diff
@@ -374,9 +374,14 @@ export function Insights(): JSX.Element {
                                     <Card style={{ marginTop: 16 }}>
                                         <BindLogic
                                             logic={trendsLogic}
-                                            props={{ dashboardItemId: null, view: activeView }}
+                                            props={{ dashboardItemId: null, view: activeView, filters: allFilters }}
                                         >
-                                            <InsightsTable />
+                                            <InsightsTable
+                                                showTotalCount={
+                                                    featureFlags[FEATURE_FLAGS.NEW_TOOLTIPS] &&
```

Comment:
Is this bugfix something we want to feature flag? 🤔

Reply:
We're not feature flagging the bug fix, `showTotalCount` is used to determine if we should display the new total count column in the table view, so I think it's fine.

Reply:
Yeah, but we just show it to those who are using the new experimental UI, even though the bug is there for everyone